src/main/java/com/google/opensesame/github/GitHubGetter.java

```diff
@@ -54,4 +54,24 @@ public static GitHub buildGitHubInterface(String clientId, String clientSecret)
       return GitHub.connectAnonymously();
     }
   }
+
+  /**
+   * Sets the GitHub API interface to the provided instance so that all of the following calls to
+   * getGitHub() will return this instance. This is intended to be used for mocking the GitHub API
+   * interface while testing and should not be elsewhere.
+   *
+   * @param mockGitHub
+   */
+  public static void mockGitHub(GitHub mockGitHub) {
+    gitHub = mockGitHub;
+  }
+
+  /**
+   * Removes the mocked GitHub API interface so that the following calls to getGitHub() will no
+   * longer return the provided instance. This is intended to be used for mocking the GitHub API
+   * interface while testing and should not be elsewhere.
+   */
+  public static void removeGitHubMock() {
+    gitHub = null;
+  }
```

Comment:
Will we ever need to call this function? I'd assume that we would only set the mock for testing purposes and that we'd be able to use the same mock throughout testing.

Reply:
That's a good point. I had imagined that the best practice with mocking would be to clean up the mock after the tests using the mock had concluded, but we do intend to use the mocks in every test that uses the GitHub getter, so maybe this should be removed?